components/e2etests/src/parser/parser_test.rs

```diff
@@ -0,0 +1,134 @@
+use super::*;
+
+#[test]
+fn locale_test_output_deser() {
+    let data = r#"
+{"lang": "en",
+ "region": "US",
+ "subtags": [["u", "hc", "buddhist"]]}
+"#;
+
+    let act_test_output: LocaleTestOutput =
+        serde_json::from_str(data).expect("cannot parse sample LocaleTestOutput");
```

Comment:
You can use `serde_json::json` macro - https://docs.serde.rs/serde_json/macro.json.html